src/components/shared/topbar/topbar.jsx

```diff
@@ -70,7 +70,7 @@ const TopBar = ({ isDarkTheme }) => (
         isDarkTheme ? 'text-gray-new-90' : 'text-gray-new-15'
       )}
     >
-      Teams switch from Aurora Serverless to Neon for better performance and 80% less costs. Read more 
+      Switch to Neon - We offer personalized migration assistance and waive all migration fees    
```

Comment:
Migration Assistance - Get expert help migrating your business workloads to Neon.